package.json

```diff
@@ -20,9 +22,16 @@
   "bugs": {
     "url": "https://github.com/eslint/eslint.github.io/issues"
   },
+  "browser": {
```

Comment:
We can move this stuff into the dev dependencies and remove them from here. I just kept the old stuff, because it worked :)